cobigen-eclipse/cobigen-eclipse/src/com/devonfw/cobigen/eclipse/workbenchcontrol/handler/GenerateJob.java

```diff
@@ -65,7 +66,9 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
     try {
       LOG.info("Initiating CobiGen...");
       monitor.beginTask("Initiating CobiGen...", 1);
-      CobiGenWrapper generator = GeneratorWrapperFactory.createGenerator(this.selection, monitor);
+      if (PostponeUtil.isTimePassed())
+        checkMonolithicConfigurationException(monitor);
```

Comment:
Is this methode just there to call the GeneratorWrapperFacroty.createGenerator with false instead of true?